server/Dockerfile.test

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+FROM node:16-alpine
+
+WORKDIR /app
+
+COPY . .
+
+RUN chown -R node:node ./*
+
+RUN npm ci
+
+RUN npm run build
+
+USER node
+
+CMD ["/bin/sh", "-c", "npm run migration:up && npm run seed:up && npm test"]

```

Comment:
Is it possible to pass some argument to specify either test or start so we don't need two nearly identical Dockerfiles?